ui/articles-aem-renderer/src/androidTest/java/org/cru/godtools/articles/aem/ArticleParseTest.java

```diff
@@ -0,0 +1,59 @@
+package org.cru.godtools.articles.aem;
+
+import android.net.Uri;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.cru.godtools.articles.aem.model.Article;
+import org.cru.godtools.articles.aem.model.Attachment;
+import org.cru.godtools.articles.aem.service.support.ArticleParser;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import static junit.framework.Assert.assertTrue;
+import static junit.framework.Assert.fail;
+
+@RunWith(AndroidJUnit4.class)
+public class ArticleParseTest {
+
+    @Test
+    public void verifyArticleParseLogic() {
+
+        try {
+            Uri url = Uri.parse("https://stage.cru.org/content/experience-fragments/questions_about_god/english.999.json");
+
+            OkHttpClient client = new OkHttpClient();
+
+            Request request = new Request.Builder()
+                    .url(url.toString())
+                    .build();
+
+            Response response = client.newCall(request).execute();
+
+            String result = response.body().string();
```

Comment:
for tests you shouldn't be relying on fetching content from the web, it makes a test very brittle. you should include the json as a file and parse that. This makes the test self-contained and not dependent on an external system. see:
https://github.com/CruGlobal/godtools-android/blob/develop/library/xml-model/src/androidTest/java/org/cru/godtools/xml/model/ManifestIT.java#L18-L23
https://github.com/CruGlobal/godtools-android/blob/develop/library/xml-model/src/androidTest/assets/tests/manifest-empty.xml